### DISCUSSION TITLE
New Icon

### DISCUSSION THREAD

**Discussion by NeKoOuO** [open-ani/animeko#125] [Ideas]
_Outdated_

**Comment by NeKoOuO** [comment]
_Outdated_

**Reply by Him188** [reply]
哈哈, 喜欢你这种自己动手解决问题的, 采用了!

**Reply by NeKoOuO** [reply]
哈哈哈不嫌棄就好

**Comment by NeKoOuO** [comment]
White verson 


![あ](https://github.com/Him188/ani/assets/111511925/cdd19087-80a2-43af-a15f-fa513df011d4)

**Comment by NeKoOuO** [comment]
Black verson

![あ](https://github.com/Him188/ani/assets/111511925/c379dc87-f967-4b20-9612-115e77747a0e)

**Comment by Him188** [comment]
可以麻烦导出一下下面两种类型的吗

一种是方形的 (不一定需要是圆角, 系统会自动裁剪)

![ic_launcher](https://github.com/Him188/ani/assets/12100985/0d732440-f8a5-430c-b7e5-0b3f77372115)

一种是圆的

![ic_launcher_round](https://github.com/Him188/ani/assets/12100985/453ca9d8-85f2-446f-a857-e7d15f31cbc1)


因为用户选择的主题不同, 他们的手机系统会选择其中一种图标展示, 如果只提供方形的图标就会导致它把正方形缩放到圆形以内...

----

另外如果你那边方便的话可以文件名改成 a.png (和 a_round.png) 吗, 这样就能直接复制进去不用我分别改名字 (Android 不支持 unicode 文件名...)

**Reply by NeKoOuO** [reply]
好我試試

**Comment by NeKoOuO** [comment]
[AppIcons_round.zip](https://github.com/Him188/ani/files/14894637/AppIcons_round.zip)

![a_round](https://github.com/Him188/ani/assets/111511925/ba7ff3f4-405f-48a3-abe9-eb21cc9c4515)

**Reply by NeKoOuO** [reply]
重改名了 這樣你就不用重複改了
[AppIcons_a.zip](https://github.com/Him188/ani/files/14894653/AppIcons_a.zip)

**Reply by Him188** [reply]
太赞了

**Comment by Him188** [comment]
![image](https://github.com/open-ani/ani/assets/12100985/38282108-4341-4f72-b89e-9ed5e858d58b)

感觉圆的版本应该把外面那个篮圈去掉会好看点

**Reply by Him188** [reply]
![image](https://github.com/open-ani/ani/assets/12100985/a25f6fab-990b-48e3-b8bc-de96e2d2ee47)

**Reply by NeKoOuO** [reply]
> ![image](https://github.com/open-ani/ani/assets/12100985/38282108-4341-4f72-b89e-9ed5e858d58b)
> 
> 感觉圆的版本应该把外面那个篮圈去掉会好看点

原本就沒有阿🤔 我設計的時候就沒有

**Reply by Him188** [reply]
草, 确实

好像是 google 桌面导致的
